src/context/ContextTypes.ts

```diff
@@ -40,48 +103,61 @@ export interface ContactID {
   FDS_ID?: string;
 }
 
-export interface Email {
+export interface Country {
+  id: CountryID;
   type: string;
-  recipients: object;
-  subject?: string;
-  textBody?: string;
-  richTextBody?: object;
+  name?: string;
 }
 
-export interface InstrumentList {
-  instruments: Instrument[];
+export interface CountryID {
+  COUNTRY_ISOALPHA2?: string;
+  COUNTRY_ISOALPHA3?: string;
+  ISOALPHA2?: string;
+  ISOALPHA3?: string;
+}
+
+export interface Currency {
+  id: CurrencyID;
+  name?: string;
+  type: string;
+}
+
+export interface CurrencyID {
+  CURRENCY_ISOCODE?: string;
+}
+
+export interface Email {
+  recipients: RecipientsObject;
+  subject?: string;
+  textBody?: string;
   type: string;
   id?: { [key: string]: string };
   name?: string;
 }
 
-export interface Instrument {
-  id: InstrumentID;
+export interface RecipientsObject {
```

Comment:
I don't like `RecipientsObject` as a name for developers to use. Names should be domain-based, not with development concepts like "object" in them.

This is probably a `QuickType` thing, and can hopefully be tweaked by playing with the command line parameters used in package.json.

If not, I think we should investigate these options:
* Try to control the name QuickType generates through its settings
* Manually change the file after generation
* Automatically change the file after generation

Reply:
I'm not hugely bothered by this as I doubt anyone will use that generated type directly. Its only exported to facilitate type checking. 

I'm also not seeing an option to fix it in the [TypeScript flow renderer in quicktype](https://github.com/quicktype/quicktype/blob/master/src/quicktype-core/language/TypeScriptFlow.ts) (the javascript name option doesn't do it. You can try it here: https://app.quicktype.io/

We may, however, wish to disable the javascript name option as it is replacing capitalized properties or those with underscores with camelcase... Pretty sure we don't want that.

Reply:
Seems like losing the `*Object` names would require at least a custom language definition - which didn't defer to the code generating the names in the ConvenienceRenderer: https://blog.quicktype.io/customizing-quicktype/

Reply:
Okay, I agree it's not a biggie, we can always add a separate issue, and release an update of the npm package only if we find a way to deal with it. I will approve.